changelog/4323.trivial.rst

```diff
@@ -0,0 +1,2 @@
+Changed `sunpy.coordinates.sun.B0` return type from `~astropy.coordinates.Angle`
```

Comment:
```suggestion
Changed :func:`sunpy.coordinates.sun.B0` return type from `~astropy.coordinates.Angle`
```

Reply:
I don't think this is necessary, the link works fine without: https://sunpy--4323.org.readthedocs.build/en/4323/whatsnew/changelog.html

Reply:
It adds the parentheses